### PR TITLE
Fixing carbon-dashboard issue-688

### PR DESCRIPTION
**Pull request by Minoli**
Fixing https://github.com/wso2/carbon-dashboards/issues/688

### CODE REVIEW

File: components/dashboards-web-component/src/designer/DashboardDesigner.css [hunk 1]
```diff
@@ -96,4 +96,46 @@
...
+
+.designer-left-menu {
+    padding: 0 !important;
+}
+
+/* dashboard designer golden layout style customizations */
+.dashboard-designer .lm_header.lm_selectable {
+    background: #192830 !important;
+    height: 37px !important;;
+}
```
Comment (erangatl) [inline components/dashboards-web-component/src/designer/DashboardDesigner.css]: There is an extra semicolon, pls remove.

File: components/dashboards-web-component/src/designer/DashboardDesigner.css [hunk 2]
```diff
@@ -96,4 +96,46 @@
...
+    background: #13222a !important;
+    border-color: #13222a !important;
+}
+
+.dashboard-designer .lm_header .lm_tab.lm_active {
+    box-shadow: none;
+    background: #13222a;
+    height: 25px;
+    margin-top: 0px;
+    border: none;
```
Comment (erangatl) [inline components/dashboards-web-component/src/designer/DashboardDesigner.css]: When the value is 0 no need to specify the unit, pls check and fix in other places as well.

Comment (Minoli) [reply components/dashboards-web-component/src/designer/DashboardDesigner.css]: fixed

File: components/dashboards-web-component/src/designer/DashboardDesigner.css (hunk 1 above)
Comment (Minoli) [reply components/dashboards-web-component/src/designer/DashboardDesigner.css]: fixed

**Pull request merged**